src/composables/useVeBAL.ts

```diff
@@ -18,8 +18,9 @@ const showRedirectModal = ref(false);
 /**
  * COMPUTED
  */
-export const isVeBalSupported = computed(
-  () => isMainnet.value || isGoerli.value
+export const isVeBalSupported = computed<boolean>(
+  () =>
+    configService.network.addresses.veBAL !== '' && POOLS.IdsMap?.veBAL !== ''
```

Comment:
I think we can simplify this, the IdsMap is just a helper map so you can access a poolId by name.
```suggestion
    configService.network.addresses.veBAL !== ''
```
Maybe we should even make the `veBAL` key on addresses optional? I don't like that we have a bunch of keys in the config which are just empty strings. If the contract doesn't exist on a network then the key shouldn't really be accessible.

Reply:
Actually it seems like it might be more trouble than it's worth as that string is used in many places that don't accept undefined, so they'd all have to be changed to `address.veBAL || ''` or have an if around them anyway.